modules/cnf-reducing-netqueues-using-nto.adoc

```diff
@@ -9,11 +9,6 @@ The Node Tuning Operator allows you to adjust the network interface controller (
 
 In real-time or low latency systems, all the unnecessary interrupt request lines (IRQs) pinned to the isolated CPUs must be moved to reserved or housekeeping CPUs.
 
-In deployments with applications that require system, {product-title} networking or in mixed deployments with Data Plane Development Kit (DPDK) workloads, multiple queues are needed to achieve good throughput and the number of NIC queues should be adjusted or remain unchanged. For example, to achieve low latency the number of NIC queues for DPDK based workloads should be reduced to just the number of reserved or housekeeping CPUs.
+For deployments with applications needing optimal throughput and low latency, it is important to adjust the NIC queue count. For example, to achieve low latency the number of NIC queues for DPDK based workloads should be reduced to just the number of reserved or housekeeping CPUs.
```

Comment:
This still doesn't make sense, the PerformanceProfile only deals with interfaces in kernel-mode.

Suggest the entire section should be removed, only keeping the first line:

`The Node Tuning Operator allows you to adjust the network interface controller (NIC) queue count for each network device. By using a PerformanceProfile, the amount of queues can reduced to the amount of `reserved` cpus.`